force-app/main/default/objects/CQ_AS_SQX_Teacher__c/fields/CQ_AS_SQX_Account__c.field-meta.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>CQ_AS_SQX_Account__c</fullName>
+    <deleteConstraint>SetNull</deleteConstraint>
+    <externalId>false</externalId>
+    <label>Account</label>
+    <referenceTo>Account</referenceTo>
+    <relationshipLabel>Teachers</relationshipLabel>
+    <relationshipName>Teachers</relationshipName>
```

Comment:
we follow naming conv in relationship name too

Reply:
Followed now